### PR TITLE
Adding support for HyperV Guest Interface Services

### PR DESCRIPTION
**Pull request by gaelcolas**
Hi, 
This PR is to add support for the Guest Services when the VM is created.

The Guest Services is what allows 'Enhanced' console similar to RDP (i.e. Copy paste).

### CODE REVIEW

File: lib/kitchen/driver/hyperv.rb [hunk 1]
```diff
@@ -187,6 +188,10 @@ def vhd_folder?
         config[:parent_vhd_folder] && Dir.exist?(config[:parent_vhd_folder])
       end
 
+      def enable_guest_services
+        @enable_guest_services ||= config[:enable_guest_services]
```
Comment (mwrock) [inline lib/kitchen/driver/hyperv.rb]: I would leave out this method and just refer to this as `config[:enable_guest_services]`

File: lib/kitchen/driver/powershell.rb [hunk 2]
```diff
@@ -113,6 +113,7 @@ def new_vm_ps
             DynamicMemoryMinBytes = #{config[:dynamic_memory_min_bytes]}
             DynamicMemoryMaxBytes = #{config[:dynamic_memory_max_bytes]}
             boot_iso_path = "#{boot_iso_path}"
+            EnableGuestServices = "#{enable_guest_services}"
```
Comment (mwrock) [inline lib/kitchen/driver/powershell.rb]: see above

**Comment by gaelcolas** [comment]
Thanks @mwrock for the feedback.
Updated the PR, how does that look now?

**Comment by mwrock** [comment]
:+1:

**Comment by usepowershell** [comment]
thanks @gaelcolas

**Pull request merged**